osu.Game/Overlays/SkinEditor/SkinEditor.cs

```diff
@@ -345,6 +342,7 @@ private void targetChanged(ValueChangedEvent<SkinComponentsContainerLookup?> tar
             changeHandler = new SkinEditorChangeHandler(skinComponentsContainer);
             changeHandler.CanUndo.BindValueChanged(v => undoMenuItem.Action.Disabled = !v.NewValue, true);
             changeHandler.CanRedo.BindValueChanged(v => redoMenuItem.Action.Disabled = !v.NewValue, true);
+            changeHandler.SaveState();
```

Comment:
The better way to do this would likely entail doing this in `SkinEditorChangeHandler` itself. Likely requires changing

https://github.com/ppy/osu/blob/6a04708a7e9801949c6c7ac7ddf6a4d7fa0835e5/osu.Game/Overlays/SkinEditor/SkinEditorChangeHandler.cs#L36

to `BindValueChanged(..., true)`. This way you won't have to duplicate this same call in the test as well.